narayana/README.md

```diff
@@ -26,6 +26,10 @@ For example, to run the VolatileStore benchmarks type the following:
 
     java  -jar ./ArjunaJTA/jta/target/benchmarks.jar com.arjuna.ats.jta.xa.performance.VolatileStoreBenchmark.* -i 1 -wi 2 -f 1 -t 2 -r 10
  
+To run the VolatileStore benchmarks failing immediately if any benchmark had experienced an unrecoverable error type the following :
```

Comment:
You need a comma around the independent clause since otherwise the sentence is difficult to parse, ie:

```
To run the VolatileStore benchmarks, failing immediately if any benchmark had experienced an unrecoverable error, type the following`

Reply:
Thank you @mmusgrov for the review.